src/pages/HomePage/styles.js

```diff
@@ -0,0 +1,88 @@
+import styled from 'styled-components';
+
+import illustration from '../../assets/img/illustration-empty-state.png';
+
+export const Container = styled.main`
+	align-items: center;
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	margin: auto;
+	max-width: 1200px;
+	min-height: 100vh;
+	padding: 1rem;
+	width: 100%;
+`;
+
+export const Empty = styled.section`
+	align-items: center;
+	display: flex;
+	flex: 1;
+	flex-direction: column;
+	justify-content: center;
+	text-align: center;
+	width: 100%;
+`;
+
+export const Illustration = styled.div`
+	background-image: url(${illustration});
+	background-position: center;
+	background-repeat: no-repeat;
+	background-size: contain;
+	height: 220px;
+	margin-top: 2rem;
+	max-width: 440px;
+	width: 100%;
+
+	@media (min-width: 576px) {
+		background-size: cover;
+	}
+`;
+
+export const Title = styled.h2`
+	font-size: 1.5rem;
```

Comment:
Take a look at `ThemeProvider` of styled-components, I think it will help you to make the style more solid.